src/staking-v3/components/data/DataList.vue

```diff
@@ -143,6 +149,44 @@ export default defineComponent({
       () => tiersConfiguration.value.numberOfSlots - dAppTiers.value.dapps.length
     );
 
+    const dappsPerTier = computed<number[]>(() => {
+      const tierCounts: { [key: number]: number } = {};
+
+      // Loop through each dApp in the list and count occurrences per tierId
+      dAppTiers.value.dapps.forEach((dapp) => {
+        const tierId = dapp.tierId!;
+        tierCounts[tierId] = (tierCounts[tierId] || 0) + 1;
+      });
+
+      // Convert the tierCounts object to a sorted array of counts
+      const sortedCounts = Object.entries(tierCounts)
+        .sort((a, b) => Number(a[0]) - Number(b[0]))
+        .map((entry) => entry[1]);
+
+      return sortedCounts;
+    });
+
+    const tokensToBeBurned = computed(() => {
+      // Calculate the sum of tokens to be burned
+      const tbb = dAppTiers.value.rewards.reduce((acc: bigint, reward: BigInt, i) => {
+        const slotsPerTier = tiersConfiguration.value.slotsPerTier[i];
+        const dappsInTier = dappsPerTier.value[i];
+        const tokensForTier =
+          ((BigInt(reward.toString()) * (BigInt(slotsPerTier) - BigInt(dappsInTier))) /
+            BigInt(slotsPerTier)) *
+          BigInt(111); // Days in a Build&Earn subperiod
```

Comment:
This will not work on Shibuya. Use `eraLengths.standardErasPerBuildAndEarnPeriod `
In general don't use magic numbers in code

Reply:
I was going to use `periodDuration.value` but this is much better.